### PR TITLE
clean up new project and edit project views

### PR DESCRIPTION
**Pull request by dkhgh**
* [x] Toggle
* [x] Cleaner inputs
* [x] Move link to audit trail to edit project view

### CODE REVIEW

**Comment by dkhgh** [comment]
before: 

screen shot 2015-01-13 at 18 34 35

After: 

screen shot 2015-01-13 at 18 35 01

File: app/assets/stylesheets/_tagger.css.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
   color: transparent;
   font-size: 16px;
   z-index: 1;
-  padding: 5px;
+  padding: 4px;
```
Comment (dkhgh) [inline app/assets/stylesheets/_tagger.css.scss]: WHOOPS THIS NEEDS TO GO

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -59,15 +60,6 @@ body {
...
 
-h1 {
-  margin-bottom: 1em;
-  text-align: center;
-
-  @include media($medium-screen) {
-    text-align: inherit;
-  }
-}
-
```
Comment (dkhgh) [inline app/assets/stylesheets/application.css.scss]: Not sure why this is deleted...
@fatboypunk wasn't this uui

Comment (fatboypunk) [inline app/assets/stylesheets/application.css.scss]: Might have been, but that was long ago I think.

**Comment by jurre** [comment]
Nice! I'll fix up the specs tomorrow

**Pull request merged**